### PR TITLE
whereis: add page

### PR DESCRIPTION
**Pull request by sudoio**
Add `whereis` command page.

### CODE REVIEW

**Comment by vojta001** [comment]
I would also provide an example without any switch, like `man ls`, because it is IMHO the most commonly used form.

**Comment by sbrl** [comment]
Looks ok to me! Can't figure out how to approve it in octodroid, so this comment will have to do ☺️

File: pages/linux/whereis.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- Locate binary, source and man pages for ssh:
+
+`whereis {{ssh}}`
+
+- Locate binary and man pages for ls:
+
+`whereis -bm {{ls}}`
+
+- Locate source of gcc and man pages for git:
```
Comment (agnivade) [inline pages/linux/whereis.md]: Somehow `-s` doesn't seem to work on my system. It works without it.
```
20:49:51-agniva-~/play/tldr-bot$whereis -s gcc -m git
gcc:
git: /usr/share/man/man1/git.1.gz
20:50:04-agniva-~/play/tldr-bot$whereis -s gcc 
gcc:
20:50:09-agniva-~/play/tldr-bot$whereis gcc 
gcc: /usr/bin/gcc /usr/lib/gcc /home/linuxbrew/.linuxbrew/bin/gcc /usr/share/man/man1/gcc.1.gz
```

Any idea why ?

Comment (sudoio) [reply pages/linux/whereis.md]: The reason is very simple. You don't have `source of gcc` on your machine.

Comment (agnivade) [reply pages/linux/whereis.md]: Ah I see. I thought that `/usr/lib/gcc` was the source. My bad, should have checked.

**Pull request merged**